pkgs/applications/audio/midi-trigger/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, gnumake, pkg-config, gcc, lv2 }:
+stdenv.mkDerivation rec {
+  pname = "midi-trigger";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "unclechu";
+    repo = "MIDI-Trigger";
+    rev = "v${version}";
+    sha256 = "sha256-tMnN8mTd6Bm46ZIDy0JPSVe77xCZws2XwQLQexDWPgU=";
+  };
+
+  nativeBuildInputs = [ gnumake pkg-config gcc ];
+
+  buildInputs = [ lv2 ];
+
+  buildPhase = ''
+    make BUILD_DIR=.
+  '';
+
+  installPhase = ''
+    mkdir -p -- "$out/lib/lv2"
+    mv -- "$pname".lv2 "$out/lib/lv2"
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/unclechu/MIDI-Trigger";
+    description =
+      "LV2 plugin which generates MIDI notes by detected audio signal peaks";
```

Comment:
Weird random line break?

Reply:
@K900 nope, it’s `nixfmt` making it fit 80 chars line length limit.

Reply:
According to some random data I just collected, it's all over the place.